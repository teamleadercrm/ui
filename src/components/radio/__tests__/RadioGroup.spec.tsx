import React from 'react';
import { render } from '@testing-library/react';
import { RadioButton, RadioGroup } from '../';

describe('Component - RadioButton', () => {
  it('renders', async () => {
    const { asFragment } = render(
      <RadioGroup value="bar">
        <RadioButton value="foo" label="Foo" />
        <RadioButton value="bar" label="Bar" />
      </RadioGroup>,
    );
    expect(asFragment()).toMatchSnapshot();
  });
});
