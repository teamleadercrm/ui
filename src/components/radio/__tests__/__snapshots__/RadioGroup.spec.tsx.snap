// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Component - RadioButton renders 1`] = `
<DocumentFragment>
  <div
    class="box"
    data-teamleader-ui="radio-group"
  >
    <label
      class="box radiobutton is-medium"
      data-teamleader-ui="radiobutton"
    >
      <input
        class="input"
        readonly=""
        type="radio"
        value="foo"
      />
      <span
        class="shape"
      />
      <span
        class="label"
      >
        <span
          class="box text text-body-compact darkest teal"
          data-teamleader-ui="text"
        >
          Foo
        </span>
      </span>
    </label>
    <label
      class="box radiobutton is-medium is-checked"
      data-teamleader-ui="radiobutton"
    >
      <input
        checked=""
        class="input"
        readonly=""
        type="radio"
        value="bar"
      />
      <span
        class="shape"
      />
      <span
        class="label"
      >
        <span
          class="box text text-body-compact darkest teal"
          data-teamleader-ui="text"
        >
          Bar
        </span>
      </span>
    </label>
  </div>
</DocumentFragment>
`;
